### PR TITLE
fix(#40): Fix parameter sent to API to filter user collection by season

### PR DESCRIPTION
**Pull request by gjariot**

### CODE REVIEW

File: src/client/actions/User.ts [hunk 1]
```diff
@@ -65,7 +65,7 @@ export default class User {
```
Comment (Roguyt) [inline src/client/actions/User.ts:65]: If you could add a TODO here to indicate that for the next major release we should rename the arg season to seasons

Comment (gjariot) [reply src/client/actions/User.ts:65]: The thing is that I don't know whether it is possible to specify two seasons? Whereas the `season` parameter name would be fine. I naively tried with commas only.

Comment (Roguyt) [reply src/client/actions/User.ts:65]: From what I know it isn't possible.

Comment (gjariot) [reply src/client/actions/User.ts:65]: Then is the TODO addition necessary? Current method signature is clearer than the API one actually!

Comment (Roguyt) [reply src/client/actions/User.ts:65]: I guess not.
I'll merge it

**Pull request merged**